src/Automation/Automation/help/Get-AzAutomationScheduledRunbook.md

```diff
@@ -71,6 +71,23 @@ Get-AzAutomationScheduledRunbook -AutomationAccountName "Contoso17" -ResourceGro
 
 This command gets all scheduled runbooks for the schedule Schedule01 in the Azure Automation account named Contoso17.
 
+### Example 4: Get Parameters given to a Scheduled Runbook
+
+The output of below command provides JobScheduleId associated with the runbook Runbk01.
+
+```powershell
```

Comment:
We cannot contain two powershell code blocks in one example. Please split them to two examples or merge the them into one code block.